### PR TITLE
Milestone 1: Finished Setup

### PR DESCRIPTION
**Pull request by nikoescobal**
- Added lib and bin directories
- Added .gitkeep and main.rb
- Configured main.rb file to be executable

### CODE REVIEW

File: bin/main.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env ruby
+puts "Hello!"
```
Comment (lekegitrepo) [inline bin/main.rb]: - **_[Optional] This should be "Hello World"_**

**Review by lekegitrepo — CHANGES_REQUESTED** [review]
## Status: Changes Required! :recycle:
### Hi @nikoescobal :clap: :clap:
#### Good job so far :+1: However, you to implement the following requirement before this project can be approved. Check my comments below :point_down:

#### 1) LINTER:
 - Enable linter for this Pull Request, and fix the rubocop errors.

#### _Reviewed by Temitayo Adeleke_

**Review by Georjane — CHANGES_REQUESTED** [review]
## Status: Changes Required 🔁 🚫
### Hi Team 🙌
Your setup for the Tic Tac Toe game project is ready but you have to set up your linters for the project before it gets approved, just like the previous Code Reviewer requested. 

Feel free to reach out to me on Slack if you have any questions, Witah Georjane 
Goodluck on your project and Happy Coding 💻

**Review by NtwaliHeritier — APPROVED** [review]
### STATUS: APPROVED✔✔
Good work team. 
Move on to the next milestone.

**Pull request merged**